docs: updates for empty dashboard state

imatwawana opened the pull request:
Updates related to dashboard empty state. Includes some updates related to the new navigation.

Do not merge until all related PRs are merged:
https://github.com/grafana/grafana/pull/65464

imatwawana committed:
empty dashboard related updates

polibb commented on docs/sources/dashboards/build-dashboards/create-dashboard/index.md:
```diff
@@ -23,18 +23,19 @@ Dashboards and panels allow you to show your data in visual form. Each panel nee
-1. Click **Add a new panel**.
-1. In the first line of the **Query** tab, click the drop-down list and select a data source.
+1. Click **Dashboards** in the left-side menu.
+1. Click **New** and select **New Dashboard**.
```
yeah, it's better as a separate point ☺️

polibb reviewed (approved):
very throughout! I love how consistent you made everything :)

imatwawana commented on docs/sources/dashboards/build-dashboards/create-dashboard/index.md:
```diff
@@ -23,18 +23,19 @@ Dashboards and panels allow you to show your data in visual form. Each panel nee
-1. In the first line of the **Query** tab, click the drop-down list and select a data source.
+1. Click **Dashboards** in the left-side menu.
+1. Click **New** and select **New Dashboard**.
+1. On the empty dashboard, click **Add visualization**.
```
```suggestion
1. On the empty dashboard, click **+ Add visualization**.
```

imatwawana commented on docs/sources/dashboards/build-dashboards/create-dashboard/index.md:
```diff
@@ -64,28 +68,30 @@ To see an example of repeating rows, refer to [Dashboard with repeating rows](ht
+1. At the top of the dashboard, click **Add** and select **Row** in the dropdown.
 
-1. On the **Add a panel** dialog box, click **Add a new row**.
+   If the dashboard is empty, you can just click the **Add row** option in the middle of the dashboard.
```
```suggestion
   If the dashboard is empty, you can just click the **+ Add row** button in the middle of the dashboard.
```

imatwawana commented on docs/sources/dashboards/build-dashboards/manage-library-panels/index.md:
```diff
@@ -40,12 +40,9 @@ Once created, you can modify the library panel using any dashboard on which it a
-1. Click the **Add a panel from the panel library** option.
+1. Click **Dashboards** in the left-side menu.
+1. Click **New** and select **New Dashboard**.
+1. On the empty dashboard, click **Import panel**.
```
```suggestion
1. On the empty dashboard, click **+ Import library panel**.
```

imatwawana commented on docs/sources/getting-started/build-first-dashboard.md:
```diff
@@ -40,15 +40,15 @@ To sign in to Grafana for the first time:
-1. On the Dashboards, click **New** and select **New Dashboard** from dropdown menu.
-1. On the dashboard, click **Add a new panel**.
+1. On the Dashboards page, click **New** and select **New Dashboard** from the dropdown menu.
+1. On the dashboard, click **Add visualization**.
```
```suggestion
1. On the dashboard, click **+ Add visualization**.
```

imatwawana commented on docs/sources/tutorials/grafana-fundamentals/index.md:
```diff
@@ -215,16 +215,17 @@ Dashboards consist of _panels_, each representing a part of the story you want y
-1. In the **Query editor** below the graph, enter the query from earlier and then press Shift + Enter:
+1. Click **Dashboard** in the left-side menu.
+1. On the Dashboards page, click **New** and select **New Dashboard** in the dropdown.
+1. Click **Add visualization**.
```
```suggestion
1. Click **+ Add visualization**.
```

imatwawana committed:
updated steps to clearly indicate button names

Some steps indicated the heading as the thing to click rather than the button text. Also some of the button text wasn't quite accurate.

imatwawana commented on docs/sources/dashboards/build-dashboards/create-dashboard/index.md:
```diff
@@ -48,9 +49,12 @@ Dashboards and panels allow you to show your data in visual form. Each panel nee
+1. Click **Save**.
 
-   Notes can be helpful if you need to revert the dashboard to a previous version.
+<!-- this part doesn't make sense for a first time dashboard creation - this interaction only happens when you update an existing dashboard
+   Notes can be helpful if you need to revert the dashboard to a previous version. -->
```
```suggestion
```

imatwawana committed:
remove incorrect instruction

chri2547 commented on docs/sources/dashboards/build-dashboards/create-dashboard/index.md:
```diff
@@ -64,28 +66,30 @@ To see an example of repeating rows, refer to [Dashboard with repeating rows](ht
+1. At the top of the dashboard, click **Add** and select **Row** in the dropdown.
 
-1. On the **Add a panel** dialog box, click **Add a new row**.
+   If the dashboard is empty, you can just click the **+ Add row** button in the middle of the dashboard.
```
```suggestion
   If the dashboard is empty, you can click the **+ Add row** button in the middle of the dashboard.
```

chri2547 reviewed (approved):
left a suggestion. Thank you!

imatwawana committed:
apply suggestion from review

Co-authored-by: Christopher Moyer <35463610+chri2547@users.noreply.github.com>

imatwawana committed:
pushing empty commit to trigger linting

imatwawana committed:
triggering linting
